src/storage/src/hummock/local_version.rs

```diff
@@ -322,9 +322,15 @@ impl LocalVersion {
     }
 
     pub fn clear_shared_buffer(&mut self) -> Vec<HummockEpoch> {
-        let cleaned_epochs = self.shared_buffer.keys().cloned().collect_vec();
+        let mut cleaned_epoch = self.shared_buffer.keys().cloned().collect_vec();
+        for (epochs, _) in self.sync_uncommitted_data.iter() {
+            for epoch in epochs {
+                cleaned_epoch.push(*epoch);
+            }
+        }
+        self.sync_uncommitted_data.clear();
```

Comment:
Also removing sync_uncommitted_data here. @hzxa21 
What about replicated_batches?

Reply:
We should also clear replicated_batches here.

Reply:
fixed